src/stories/theme/_typearound.css

```diff
@@ -0,0 +1,101 @@
+/* stylelint-disable selector-max-compound-selectors,
+   selector-combinator-space-before, indentation */
+
+@import '@ckeditor/ckeditor5-theme-lark/theme/mixins/_focus.css';
+
+.ck .ck-widget {
+  &.ck-widget_selected {
+    box-shadow: var(--ck-focus-outer-shadow), 0 0;
```

Comment:
Is `typearound` styling happening in this PR? If so, there are a number of things that should be updated. Otherwise, we can punt for now.
- no hover state
- no selected border style
- box shadow moved to the interior component with border-radius applied

Reply:
I've updated this to match the new designs 
![image](https://user-images.githubusercontent.com/4030377/106319495-62280000-6226-11eb-9bcc-59849a8cd1e3.png)

The focus-ring changes are scoped only to the `horizontal-line` plugin to allow image resizing and table functionality to work as expected.

Reply:
Based on @jzempel 's comment in Abstract, I've added in [margin specs](https://share.goabstract.com/38976020-8ffc-4f08-9005-a099cdd964c2?collectionLayerId=dd48c42d-e029-4787-b4ff-fec8978b5e90&mode=design), and the focus container is slightly [smaller in height at 12px](https://share.goabstract.com/38976020-8ffc-4f08-9005-a099cdd964c2?collectionLayerId=a2d615e0-ebb5-4bd2-b4b8-43e7789f1e2b&mode=design)

Reply:
Ugh, I had a stale version of this PR open when I merged. I'll bring in this feedback and the other spacing requirements into the next PR. This will also include moving the toolbar to its correct bottom position.